### PR TITLE
Don 9 userprofile

### PR DESCRIPTION
**Pull request by Jacobjeevan**
- Changed to findByIdAndUpdate (as a transaction to avoid race conditions in future)
- Slightly changed middleware in app
	- Ensured that both req is updated as well (when changes are made to user/agency)

### CODE REVIEW

**Comment by gnorbsl** [comment]
can you pull the latest master and merge it with your project? thats were all the conflicts come from we had a piush to master like 20 min ago

**Comment by gnorbsl** [comment]
nevermind :D

File: routes/users.js [hunk 1]
```diff
@@ -222,6 +221,11 @@ router.post('/signup', signupValidationRules(), validate, async (req, res) => {
   if (candidate) {
     return sendError(res, 409, 'This email is already taken. Try another');
   } else {
+    if (!password) return sendError(res, 206, { message: 'Password missing' });
```
Comment (gnorbsl) [inline routes/users.js:224]: this should not be in here

Comment (gnorbsl) [reply routes/users.js:224]: validation is now handled elsewere and I removed it a while ago

**Review by gnorbsl — CHANGES_REQUESTED** [review]
where do all the view file changes come from?

File: routes/users.js (hunk 1 above)
Comment (Jacobjeevan) [reply routes/users.js:224]: :thinking: It was from master hmm..

Comment (gnorbsl) [reply routes/users.js:224]: if you compare with current master they are not there

**Pull request closed without merging**